src/main/java/ca/on/oicr/gsi/dimsum/service/filtering/QCStatusSort.java

```diff
@@ -0,0 +1,46 @@
+package ca.on.oicr.gsi.dimsum.service.filtering;
+
+import java.util.Comparator;
+import java.util.List;
+import ca.on.oicr.gsi.cardea.data.Sample;
+
+public class QCStatusSort implements Comparator<Sample> {
+
+    private boolean reverseOrder;
+
+    public void setReverseOrder(boolean reverseOrder) {
+        this.reverseOrder = reverseOrder;
+    }
```

Comment:
There's no need to handle reversing - The `Comparator` interface has a default methods to do this